### ISSUE TITLE
Advice:Add an option for premuim players not to change their offline UUID 2.

### ISSUE THREAD

**Issue by Chaostarts** [games647/FastLogin#1234]
### Is your feature request related to a problem? Please describe.

I see that there is a permuim-uuid option configured to allow premuim players to not change their uuid. 
However, on servers where this option was previously enabled, premuim players will lose data after enabling it. 
Since premuim players can keep their premuim-uuids, why can't we add a setting to allow premuim players to keep their offline uuids? This would make it easier for the server to open later without losing data.

### Platform

All / Shared

### Relevance

- [X] I checked for existing tickets -
If there are, please vote them with a thumbs reaction and not create new ones

**Comment by Clyde6790pGIT** [comment]
I believe you can migrate their playerdata by finding their offline uuid using this tool:
https://minecraft-serverlist.com/tools/offline-uuid and then go to your server and then go into world and then playerdata, and then after that find their offline uuid and then rename it into their new uuid by finding it in https://mcprofile.io/

**Comment by Chaostarts** [comment]
> I believe you can migrate their playerdata by finding their offline uuid using this tool: https://minecraft-serverlist.com/tools/offline-uuid and then go to your server and then go into world and then playerdata, and then after that find their offline uuid and then rename it into their new uuid by finding it in https://mcprofile.io/

Yes, I manually migrate player data, but I hope the plugin can add such a feature
